src/containers/our-ecosystem/index.jsx

```diff
@@ -30,57 +30,56 @@ const OurEcosystem = ({ space, className }) => {
                     <div className="col-lg-4 col-md-6 col-sm-6 col-12">
                         <div className="rn-service-one-our-ecosystem color-shape-7">
                             <div className="inner">
-                                <div className="thumbnail">
+                                <div className="thumbnail text-center">
                                     <Image
                                         src="https://rumsan.nyc3.cdn.digitaloceanspaces.com/esatya/cva-ecosystem-banner.jpg"
                                         alt="cva-ecosystem-banner"
-                                        width={489}
-                                        height={366}
+                                        style={{
```

Comment:
Why did you change this width and height? Lets use 'Image' from "next/image"

Reply:
I was not guided  to use manager.rumsan.com  so i kept the image in public  and need to use the react/bootstrap instead of next/image as the image were not  rendering the  as  the width and height  mentioned

Reply:
@karun-rumsan Now that you know. Get access to manager.rumsan.com from mani and make the required changes.

Lets try to maintain uniformity in code.